.github/workflows/tps-check-lock.yml

```diff
@@ -0,0 +1,12 @@
+name: Check for moratorium
+
+on: [workflow_dispatch, workflow_call]
+
+jobs:
+  moratoriumCheck:
+    runs-on: ubuntu-latest
+    environment: ChangeManagement
+    steps:
+      - env:
+          TPS_API_TOKEN: ${{ secrets.TPS_API_TOKEN_PARAM }}
+        run: ./scripts/release/tps_check_lock cli ${{ github.sha }}
```

Comment:
I wonder if `github.sha` is available in this context, or if it would need to be passed through as a `env` or workflow param?

Reply:
That's a good point, thank you. You had originally added the checkout action to record release workflow for this purpose, we can add it here, too.